client/wfm/src/result/StepResult.ts

```diff
@@ -0,0 +1,17 @@
+export interface StepResult {
```

Comment:
ah, I hadn't updated those yet because this was still WIP, so I thought you were talking about something else :+1: